recipe/shopware6.php

```diff
@@ -0,0 +1,59 @@
+<?php
+namespace Deployer;
+
+set('repository', 'git@github.com:shopware/production.git');
```

Comment:
I believe this should be per project-based?

Reply:
The URL given is the standard-production template for shopware. It is of course required to fork that repository and use an custom URL within deploy.php
I thought it would be handy if it works even without it....

Reply:
But it will be more likely overrided by init scriot.

Reply:
> But it will be more likely overrided by init scriot.

Right. And that would be totally ok. Anyway - I got you. So should I rather remove that?